app/forms/english_gcse_grade_attribute_builder.rb

```diff
@@ -0,0 +1,237 @@
+module EnglishGcseGradeAttributeBuilder
```

Comment:
Before you review this module, I moved the code from candidate interface so I could reuse into support interface.
I see that this module needs some refactoring but I decided just to move the code. **I was reluctantly to do any changes  on this code to avoid adding any bugs to candidate interface**. Because we are firefighting the current situation, I wanted to focus on fixing the support onlu as I am trying not to make things worst 😅. Let me know your thoughts.